master/reference/public-cloud/azure.md

```diff
@@ -11,10 +11,10 @@ While Azure does not support {{site.prodname}} networking, you can use
 - **Azure user-defined routes**: This option provides networking without overlays.
   Disable {{site.prodname}} networking by setting `CALICO_NETWORKING_BACKEND` to `none` 
   in `{{site.nodecontainer}}`. (Also called "policy-only mode".) Refer to 
-  [Configuring calico/node](../../../../reference/node/configuration) and [Azure user-defined routes](#azure-user-defined-routes) for more information. {% if include.orch == "Kubernetes" %}You can
-  find a sample manifest in the [Kubernetes API datastore](../../installation/hosted/kubernetes-datastore/#policy-only) installation guide.
+  [Configuring calico/node](../node/configuration) and [Azure user-defined routes](#azure-user-defined-routes) for more information. {% if include.orch == "Kubernetes" %}You can
```

Comment:
Does this work? I don't think this file is included - it's a standalone doc.

Reply:
It probably should always say k8s anyway - I think that while strictly speaking this could apply to other orchestrators, it's operationally complex enough we'd only want to recommend it for k8s since it does it for you.

Also, we might just want to drop this and recommend using the Azure CNI plugin eventually, so it doesn't matter _too_ much what we've got here for now.